### PR TITLE
Hot reload beginning to work

### PR DESCRIPTION
**Pull request by dartharaknus**
does a full refresh so far, might be a bug, but it is better than no refresh at all.

### CODE REVIEW

**Comment by dartharaknus** [comment]
Fixes #451

**Comment by dartharaknus** [comment]
Wait for me to push the filter hero commit too.

**Comment by dartharaknus** [comment]
@howardchung This is now ready to be merged

File: src/components/Heroes/Heroes.jsx [hunk 1]
```diff
@@ -43,7 +60,23 @@ class Heroes extends Component {
...
       </div>);
     }
-    return <HeroesIndex heroes={createHeroList(heroes)} />;
+
+    return (
+      <div>
+        <h1 className={style.Header}>Heroes</h1>
+        <div className={style.SearchBar}>
+          <TextField
+            hintText={'Filter Heroes'}
```
Comment (howardchung) [inline src/components/Heroes/Heroes.jsx]: externalize string

File: src/components/Heroes/Heroes.jsx [hunk 2]
```diff
@@ -43,7 +60,23 @@ class Heroes extends Component {
         </Tabs>
       </div>);
     }
-    return <HeroesIndex heroes={createHeroList(heroes)} />;
+
+    return (
+      <div>
+        <h1 className={style.Header}>Heroes</h1>
```
Comment (howardchung) [inline src/components/Heroes/Heroes.jsx]: externalize

**Comment by howardchung** [comment]
Could you also reduce the hero image size: #417

**Comment by dartharaknus** [comment]
Updated

File: src/components/Heroes/Heroes.jsx [hunk 3]
```diff
@@ -66,7 +67,7 @@ class Heroes extends Component {
         <h1 className={style.Header}>Heroes</h1>
```
Comment (howardchung) [inline src/components/Heroes/Heroes.jsx]: one more string here

**Pull request merged**